simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp

```diff
@@ -850,6 +882,25 @@ void SimplePlanningSimulator::publish_tf(const Odometry & odometry)
   tf_msg.transforms.emplace_back(std::move(tf));
   pub_tf_->publish(tf_msg);
 }
+
+void SimplePlanningSimulator::publish_actuation_status()
+{
+  if (vehicle_model_type_ != VehicleModelType::ACTUATION_CMD) {
+    return;
+  }
+  const auto model = std::dynamic_pointer_cast<SimModelActuationCmd>(vehicle_model_ptr_);
```

Comment:
The use of `dynamic_pointer_cast` should be avoided as much as possible. In this case, you can add a  getActuationStatus() in a base class with returning zero by default.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8415/commits/ed78398b7705c83460c9089f2aef53019f552e9f

Reply:
>  returning zero by default.

I return not zoro but null